k8s/namespaces/rd/rd-location-ref-api-integration/rd-location-ref-api-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/location-ref-api:prod-7a3ef4a-20210825101927
+      image: hmctspublic.azurecr.io/rd/location-ref-api:prod-7a3ef4a-20210825101927   #{"$imagepolicy": "flux-system:rd-location-ref-api-integration"}
```

Comment:
Can you change this policy to `rd-location-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-location-ref-api-integration`

Reply:
Image policy/repo now updated.